### DISCUSSION TITLE
Setting collation for a specific connection

### DISCUSSION THREAD

**Discussion by davideio87** [dbeaver/dbeaver#11137] [Q&A] [answered]
#### System information: 
- Operating system Windows 10
- DBeaver version:7.2

Hello
I have a problem with the default collation in my Dbeaver editors.
The DB I'm working on is a Mysql  8.0.19

I have several views created via PMA, whenever I try to query them via Dbeaver using Where conditions on string, I have errors.
Let's semplify the most:

Create a view on PMA
```
CREATE VIEW v_test AS 
SELECT 'A' AS col

```

Try to query on Dbeaver

`select * from v_test WHERE col='A'`

I get:

SQL Error [1267] [HY000]: Illegal mix of collations (utf8mb4_unicode_ci,COERCIBLE) and (utf8mb4_0900_ai_ci,COERCIBLE) for operation '='

How can I set to use utf8mb4_unicode_ci collation on Dbeaver, for this specific connection?
I found only the String Encoding option:

![dbeaver](https://user-images.githubusercontent.com/64830100/103099265-132af180-460e-11eb-84a5-ec743a64ef02.PNG)

Thanks!

**Comment by kseniiaguzeeva** [comment]
Please try to set Default charset and Collation for the schema with the view. Open the schema in the Object Editor for this:
![Screenshot 2020-12-25 at 10 12 39](https://user-images.githubusercontent.com/51405061/103124759-d4427d80-4699-11eb-88cd-c5e54b78d817.png)

**Comment by davideio87** [comment]
Hi
thanks for the reply
From what I understand, your proposal will change the DB collation, not the connection collation, am I wrong?

**Comment by kseniiaguzeeva** [comment, marked as answer]
I gave you a wrong advice. Please try to change ConnectionCollation in MySQL Driver Properties. It should help.
![Screenshot 2021-01-06 at 08 33 02](https://user-images.githubusercontent.com/51405061/103733083-f4f3c700-4ff9-11eb-9bc0-0eb02a32d9f5.png)
